SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/SpatialStatics.h"
+#include "GeneralProjectSettings.h"
+#include "SpatialConstants.h"
+#include "SpatialGDKSettings.h"
+#include "Utils/ActorGroupManager.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialStatics);
+
+bool USpatialStatics::SpatialNetworkingEnabled()
+{
+	return GetDefault<UGeneralProjectSettings>()->bSpatialNetworking;
+}
+
+bool USpatialStatics::SpatialOffloadingEnabled()
+{
+	return SpatialNetworkingEnabled() && GetDefault<USpatialGDKSettings>()->bEnableOffloading;
+}
+
+bool USpatialStatics::ActorGroupAssociationMatchesWorkerType(const AActor* Actor)
+{
+	if (Actor == nullptr)
+	{
+		return false;
+	}
+
+	UActorGroupManager* ActorGroupManager = UActorGroupManager::GetInstance();
+
+	if (UWorld* World = Actor->GetWorld())
+	{
+		FString WorkerType = World->GetGameInstance()->GetSpatialWorkerType();
+
+		FName ActorGroup = ActorGroupManager->GetActorGroupForClass(Actor->GetClass());
```

Comment:
```suggestion
		const FName& ActorGroup = ActorGroupManager->GetActorGroupForClass(Actor->GetClass());
```

Reply:
Is there any reason to query the ActorGroup at all in this method?

Reply:
Not from a functional perspective, only to just give a more specific error in the failure case, whether the method was unable to find an actor group or whether it found the actor group but couldnt find the worker association. Not really opinionated, happy to change